pkgs/games/julius/default.nix

```diff
@@ -18,16 +25,42 @@ stdenv.mkDerivation rec {
     hash = "sha256-I5GTaVWzz0ryGLDSS3rzxp+XFVXZa9hZmgwon/6r83A=";
   };
 
-  nativeBuildInputs = [ cmake ];
-  buildInputs = [ SDL2 SDL2_mixer libpng ];
+  patches = [
+    # This disables the broken fixup_bundle and
+    # also disables unit tests.
+    ./darwin-fixes.patch
+  ];
+
+  postPatch = lib.optionalString stdenv.isDarwin ''
+    substituteInPlace src/platform/sound_device.c \
+      --replace-fail "#include \"SDL_mixer.h\"" "#include \"SDL2/SDL_mixer.h\""
```

Comment:
Can we add the correct ifdefs here and make that unconditional?

Reply:
Added this as part of the patch itself, using ifdefs as suggested